libs/blocks/tag-selector/tag-selector.css

```diff
@@ -21,6 +21,7 @@
   width: 110px;
   margin-bottom: 5px;
   cursor: pointer;
+  right: 109px;
```

Comment:
I'm guessing you're just dealing with some fixed dimensions and responsiveness is not a big concern for such a tool, right? Otherwise, I might have suggested to make its parent, `.tag-preview-container`, relative and position the button relative to it.